### PR TITLE
[WIP] precise driver

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

**Review by fmessmer — CHANGES_REQUESTED** [review]
thanks...I will continue based on this...
I think we still need to support kinetic-devel fir the time being...

File: precise_driver/package.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>precise_driver</name>
+  <version>0.0.0</version>
+  <description>The precise_driver package</description>
+
+  <maintainer email="benjamin.maidel@mojin-robotics.de">Benjamin Maidel</maintainer>
+  <author email="benjamin.maidel@mojin-robotics.de">Benjamin Maidel</author>
+
+  <license>private</license>
```
Comment (floweisshardt) [inline precise_driver/package.xml:10]: should be Apache 2

File: precise_description/package.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <name>precise_description</name>
+  <version>0.0.1</version>
+  <description>
+      URDF Description for Precise Flex 400
+  </description>
+
+  <author>Benjamin Maidel</author>
+  <maintainer email="benjamin.maidel@mojin-robotics.de">Benjamin Maidel</maintainer>
+
+  <license>BSD</license>
```
Comment (floweisshardt) [inline precise_description/package.xml:12]: should be Apache 2

**Comment by fmessmer** [comment]
as discussed: I modified the default branch (and the target branch of this PR) to `kinetic-devel` - as the HW is still kinetic for the time being...but code should still support both distros

**Pull request closed without merging**